swervelib/redux/HeliumCanCoderFactoryBuilder.java

```diff
@@ -43,24 +43,22 @@ private static class EncoderImplementation implements AbsoluteEncoder {
         private final Canandcoder encoder;
 
         private EncoderImplementation(Canandcoder encoder) {
-            this.encoder = encoder;
+            this.encoder = encoder; 
+            Settings settings = encoder.getSettings();
```

Comment:
We don't need to call encoder.getSettings() - we can just instantiate a new Settings() object. See their javadocs for how the set... stuff works